tdrs-frontend/src/components/NavItem/NavItem.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+/**
+ *
+ * @param {string} pathname - The full current url path.
+ * @param {string} tabTitle - The text that you want displayed
+ * on the navigation tab.
+ * @param {string} href - The path for where the user should
+ * be redirect on click.
+ */
+function NavItem({ pathname, tabTitle, href }) {
+  return (
+    <li className="usa-nav__primary-item">
+      <a
+        href={href}
+        key="welcome"
```

Comment:
@hilvitzs I think the key should be unique: https://reactjs.org/docs/lists-and-keys.html